.github/workflows/prepare-new-release.yml

```
@@ -34,116 +34,117 @@
+          set -euo pipefail
+          toml_current_version=$(grep '^version' Cargo.toml | sed 's/version = "\(.*\)"/\1/')
+          expect_version="${VERIFY_VERSION}-dev"
+          if [[ "$toml_current_version" != "$expect_version" ]]; then
+            echo "::error title=bad version::Expected version $expect_version does not match current version $toml_current_version."
+            exit 1
+          fi
+        env:
+          VERIFY_VERSION: ${{ github.event.inputs.base_ref_version }}
+
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)